docs/kb/api-version-4/prospects.md

```diff
@@ -269,11 +267,11 @@ JSON Example:
             "first_name": "New first name",
             "last_name": "New last name"
         },
-        "some@email.com": {
+        "1337": {
```

Comment:
It might be good to point out that for batchCreate, no IDs are necessary, so a valid input for batchCreate is:
```json
{
    "prospects": [
        {"email":"some@email.com","first_name":"First name","last_name":"Last name"},
        {"email":"some@email.com","first_name":"First name","last_name":"Last name"},
        {"email":"some@email.com","first_name":"First name","last_name":"Last name"}
    ]
}
```